README.adoc

```diff
@@ -61,46 +64,59 @@ It could be omitted as the title already says that it is an experimental feature
 * Avoid the phrase “It is recommended to” when proposing a solution.
 * Capitalize all Cypher keywords, i.e., `MATCH` rather than `match` or `Match`.
 
+[[error-notification-components]]
+=== Error and notification components
 
-=== Status code elements
+Each error and notification comprises the following components, some of which are documented only here in this repo.
 
-Each status code comprises the following components, some of which are documented only here in this repo.
-
-[options="header",cols="h,2,2",]
+[options="header",cols="h,2,2,2"]
 |===
 | Element
+| More information
 | Notification
 | Error
 
-| Status code (`Neo.[Class].[Subclass].[Name]`)
-| `Neo.ClientNotification.Statement.CartesianProduct`.
-| `Neo.DatabaseError.Statement.RemoteExecutionFailed`.
+| StatusCode (`Neo.[Class].[Subclass].[Name]`)
+| See <<statuscode-syntax, StatusCode syntax>>.
+| For example, `Neo.ClientNotification.Statement.CartesianProduct`.
+| For example, `Neo.DatabaseError.Statement.RemoteExecutionFailed`.
 
-| Title/Description of the status code
-| `This query builds a cartesian product between disconnected patterns.`
-| `The database was unable to execute a remote part of the statement.`
+| Notification title/ Error description (Will be removed in 6.0)
+| See <<notification-title-error-description, Notification title/Error description>>.
+| For example, `This query builds a cartesian product between disconnected patterns.`
+| For example, `The database was unable to execute a remote part of the statement.`
 
 | SeverityLevel
+|
 | One of `INFORMATION`\|`WARNING`
 m| ERROR
 
 | Category
+|
 | One of `DEPRECATION`\|`HINT`\|`UNSUPPORTED`\|`UNRECOGNIZED`\|`SECURITY`\|`GENERIC`\|`PERFORMANCE`\|`TOPOLOGY`
 | N/A
 
-| Description/Message
+| Notification description (Will be removed in 6.0)
+| See <<notification-description-error-message, Notification description>>.
+| Contains the main information, such as the problem, cause, consequences, a simple solution, or a reference to the Status Codes documentation.
+| N/A
+
+| Message (A new component that will replace the notification title and description, and the error description in 6.0)
```

Comment:
Well, this is almost correct. Errors do have messages already today but we plan to update those in 6.0